back/database/migrations/2025_05_16_060711_create_furniture_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('furniture', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->text('image_url');
+            $table->text('detail');
+            $table->decimal('price');
+            $table->id('category_id');
+            $table->integer('stock');
+            $table->timestamps();
+        });
+    }
```

Comment:
```suggestion
        Schema::create('furniture', function (Blueprint $table) {
            $table->id();
            $table->string('name');
            $table->text('image_url');
            $table->text('detail');
            $table->unsignedInteger('price');
            $table->unsignedBigInteger('category_id');
            $table->unsignedInteger('stock');
            $table->timestamps();
        });
    }
```


| カラム型             | 説明                         |
|----------------------|------------------------------|
| unsignedBigInteger   | 符号なしの大きな整数         |
| string               | 255文字までの可変長文字列     |
| text                 | 長文テキスト                  |
| unsignedInteger      | 符号なしの整数               |
| timestamps           | 作成日時と更新日時の自動記録 |